client/src/app/management/components/meeting-edit/meeting-edit.component.html

```diff
@@ -59,7 +59,7 @@ <h2 *ngIf="!isCreateView">{{ editMeetingLabel | translate }}</h2>
                 formControlName="guest_ids"
                 [multiple]="true"
                 placeholder="{{ 'Participants' | translate }}"
-                [inputListValues]="members"
+                [inputListValues]="allUsers"
```

Comment:
Need to be fixed or marked as TODO:
The server does only accept members of the current committee to be set here. At least this is the case during creating, during update the server (wrongly) accepts any ID here.

Anyhow, this list should only contain the members of our committee